clang/lib/DPCT/CallExprRewriter.cpp

```diff
@@ -2118,6 +2119,24 @@ createInsertHeaderFactory(
   return createInsertHeaderFactory(Header, std::move(Input));
 }
 
+std::pair<std::string, std::shared_ptr<CallExprRewriterFactoryBase>>
+createRemoveCubTempStorageFactory(
```

Comment:
if it is cub specific,  then how about put them into a seperate cub util file?

Reply:
this function just a wrapper, and just used in CallExprRewriter.cpp, the main function is implemented in CubRule.cpp(or CUBAPIMigration.cpp later)